packages/shared/lib/core/nfts/constants/default-nft-entry-key.constant.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_NFT_ENTRY_KEY = 'data'
```

Comment:
i would rename it to something more meaningful like 
```suggestion
export const DEFAULT_NFT_FEATURE_ENTRY_KEY = 'data'
```